nativelib/src/main/resources/scala-native/gc/shared/Parsing.c

```diff
@@ -50,17 +54,52 @@ size_t Parse_Size_Or_Default(const char *str, size_t defaultSizeInBytes) {
     return defaultSizeInBytes;
 }
 
+const char *get_defined_or_env(const char *envName) {
+    if (envName == NULL)
+        return NULL;
+
+    const char *env = getenv(envName);
+    if (env != NULL)
+        return env; // override with env var
```

Comment:
Perhaps '// environment overrides compile time'

Reply:
Ok, I will change this.